appveyor/install-miniconda.ps1

```diff
@@ -168,9 +168,6 @@ python --version
 checkLastExitCode
 
 # CORE DEPENDENCIES
-# Remove pytest version limitation once
-# https://github.com/astropy/astropy/pull/6419 is solved
-Copy-Item ci-helpers\appveyor\pinned ${env:PYTHON}\envs\test\conda-meta\pinned

```

Comment:
I would keep this as a placeholder (but remote the 2 comment lines above), and just remove the pytest line from that file

Reply:
I added a comment that pins should be added to this file. The pytest bin is now also removed from the file. Not sure how well git treats empty files though. I never tried that.